### PR TITLE
Adds logging and tests to meter toast iframe

### PR DESCRIPTION
**Pull request by elijahsoria**
Adds the ability to log from MeterToastApi and adds log events for ACTION_METER_TOAST_CLOSED_BY_ARTICLE_INTERACTION, IMPRESSION_METER_TOAST, and EVENT_OFFERED_METER. Adds tests to check that logging is being done correctly.

### CODE REVIEW

File: src/runtime/meter-toast-api.js [hunk 1]
```diff
@@ -43,21 +44,31 @@ export class MeterToastApi {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    const iframeArgs = this.activityPorts_.addDefaultArguments(
+      feArgs({
+        isClosable: true,
+        publicationId: deps.pageConfig().getPublicationId(),
```
Comment (mborof) [inline src/runtime/meter-toast-api.js:50]: Default arguments populates some of these other values for you.

File: src/runtime/meter-toast-api.js [hunk 2]
```diff
@@ -43,21 +44,31 @@ export class MeterToastApi {
...
     this.activityIframeView_ = new ActivityIframeView(
       this.win_,
       this.activityPorts_,
       feUrl('/metertoastiframe'),
-      feArgs({
-        publicationId: deps.pageConfig().getPublicationId(),
-        productId: deps.pageConfig().getProductId(),
-        hasSubscriptionCallback: deps.callbacks().hasSubscribeRequestCallback(),
-      }),
+      iframeArgs,
```
Comment (mborof) [inline src/runtime/meter-toast-api.js:60]: this.activityPorts_.addDefaultArguments({
  isClosable: true,
  hasSubscriptionCallback: deps.callbacks().hasSubscribeRequestCallback(),
})

File: src/runtime/meter-toast-api.js (hunk 1 above)
Comment (elijahsoria) [reply src/runtime/meter-toast-api.js]: Good catch!

File: src/runtime/meter-toast-api.js (hunk 2 above)
Comment (elijahsoria) [reply src/runtime/meter-toast-api.js:58]: Added this with feArgs in between.

Comment (mborof) [reply src/runtime/meter-toast-api.js:58]: drop feArgs.  Default args takes care of it

Comment (elijahsoria) [reply src/runtime/meter-toast-api.js:56]: Done!

File: src/runtime/meter-toast-api.js [hunk 3]
```diff
@@ -43,21 +44,27 @@ export class MeterToastApi {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    const iframeArgs = this.activityPorts_.addDefaultArguments({
```
Comment (mborof) [inline src/runtime/meter-toast-api.js:47]: As a followup on this, there is probably some central guidance that was followed when creating this initially?  Can we update that guidance?

**Review by mborof — APPROVED** [review]

File: src/runtime/meter-toast-api.js (hunk 3 above)
Comment (elijahsoria) [reply src/runtime/meter-toast-api.js:47]: I think Chen copied a previous class that was using feArgs without addDefaultArguments. Here is an example: https://github.com/subscriptions-project/swg-js/blob/c51bdb5af989317b08922d649441f77e91be49e8/src/runtime/login-notification-api.js

**Pull request merged**